README.md

```diff
@@ -254,6 +254,28 @@ yarn stats
 
 ### Settings
 
+#### Organization Configuration
+
+Organization configuration can be generated by running:
```

Comment:
I think we should merge this section with https://github.com/openwisp/openwisp-wifi-login-pages/blob/e5792c332c4758c5391cbcbbf6366c2f1aa90b72/README.md#add-organization-configuration.